src/sentry/models/apiapplication.py

```diff
@@ -66,6 +67,20 @@ class ApiApplication(Model):
     privacy_url = models.URLField(null=True)
     terms_url = models.URLField(null=True)
 
+    # API applications can be tied to an untrusted organization.
+    #
+    # This represents several security risks. A phishing risk and cross organization access for
+    # the integrator. Before creating an access_token for a user, verify the user has access to
+    # the organization defined on the application. Restrict the access_token to only access the
+    # organization defined on the application. Cross-organization access is not tolerable
+    # regardless of the token's scope.
+    organization_id = HybridCloudForeignKey(
```

Comment:
How will partners create an ApiApplication for each organization? This same schema change was recently proposed for another project, and we found an alternative solution through both sentryapps (which have organization level installations already).

Reply:
@markstory this was a mistake. I didn't mean to add it to application, already done the model changes in api application that I needed to in https://github.com/getsentry/sentry/pull/76974  🤦‍♀️ here's the spec showing the flow and model changes: https://www.notion.so/sentry/Laravel-Partnership-For-Existing-Orgs-371b6bd1d16c447ea666e533c32e503e

Let me update and re-request review.